internal/cli/root.go

```diff
@@ -119,6 +119,7 @@ func Execute() {
 	rootCmd.AddCommand(brandingCmd(cli))
 	rootCmd.AddCommand(rolesCmd(cli))
 	rootCmd.AddCommand(ipsCmd(cli))
+	rootCmd.AddCommand(actionsCmd(cli))
```

Comment:
nit: I'll put this close to 	`rootCmd.AddCommand(rulesCmd(cli))`

Reply:
You're right, that determines the order commands will show up on the help.

Reply:
Changed in https://github.com/auth0/auth0-cli/pull/316/commits/2e653f5e2bc005ce8cd614c8243f4bd3199b992f.